examples/cloudml-document-ai-patents/config.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Config file for Document AI patents example training and prediction pipelines.
+
+# The project the Document AI pipeline will be built and hosted in.
+# TODO(michaelsherman): Make sure no line is past 80 characters (per bash style)
+pipeline_project:
+  project_id: # TODO(user): Set to your Project ID.
+  region: 'us-central1' # Currently (Aug 2019) only "us-central1" supported.
+  demo_sample_data: # TODO(user): GCS location of sample patents to process for demo
```

Comment:
These two config parameters should be cleaned up. Maybe even under a different header than pipeline_project? I dunno, that may be more confusing. But something about "demo" seems off, maybe "gcs_prediction_data_folder" and "predictions_dataset". Think about it a bit.

These comments should also end with periods?

have a better name that doesn't have "demo". Maybe something like "predictions_folder" or "folder_to_predict" or "